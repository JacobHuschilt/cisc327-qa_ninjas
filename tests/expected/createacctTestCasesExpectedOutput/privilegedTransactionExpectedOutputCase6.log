Error! You can only create accounts in agent mode.
Error! Amount input is not valid.