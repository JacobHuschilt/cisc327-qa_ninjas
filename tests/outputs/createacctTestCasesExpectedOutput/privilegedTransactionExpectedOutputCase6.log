Error, privileged transaction failure!  Can't withdraw in ATM mode.