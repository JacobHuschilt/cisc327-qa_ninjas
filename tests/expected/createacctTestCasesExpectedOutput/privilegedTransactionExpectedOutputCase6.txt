NEW 9900001 000 0000000 Kathy Mill
EOS 0000000 000 0000000 ***